plugins/inventory/aws_ec2.py

```diff
@@ -51,6 +51,22 @@
               - Available filters are listed here U(http://docs.aws.amazon.com/cli/latest/reference/ec2/describe-instances.html#options).
           type: dict
           default: {}
+        include_filters:
+          description:
+              - A list of filters. Any instances matching at least one of the filters are included in the result.
+              - Available filters are listed here U(http://docs.aws.amazon.com/cli/latest/reference/ec2/describe-instances.html#options).
+              - Every entry in this list triggers a search query. As such, from a performance point of view, it's better to
+                keep the list as short as possible.
+          type: list
+          default: []
+        exclude_filters:
+          description:
+              - A list of filters. Any instances matching one of the filters are excluded from the result.
```

Comment:
Needs to explain if include or exclude takes priority.  From the code it looks like exclude takes priority over include.

Reply:
Yes,g ood point.

Reply:
agreed, if i put like 

```
hostnames:
- tag:Name
  - my_host 
```

but that instance `my_host` was also under exclude_filters what would happen?

Reply:
Exclude prevails, you would get an empty list.